spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -92,11 +92,8 @@ def argv(self):
                 self.set_conf('default', True, section='main_interpreter')
                 self.set_conf('custom', False, section='main_interpreter')
 
-        # Part of spyder-ide/spyder#11819
-        is_different = is_different_interpreter(pyexec)
```

Comment:
It seems this was the only use case of the function `is_different_interpreter`, so you should be able to remove it as part of this PR too.